src/main/java/seedu/souschef/model/AppContent.java

```diff
@@ -73,4 +99,27 @@ public boolean equals(Object other) {
     public int hashCode() {
         return recipes.hashCode();
     }
+
+
+    //healthplan level operations
+    public UniqueList<HealthPlan> getHealthPlans() {
+        return healthPlans;
+    }
+
+
+    @Override
+    public ObservableList<HealthPlan> getHealthPlanList() {
```

Comment:
Rename to getObservableHealthPlanList().
Also shift this under getObservableRecipeList()